src/installer/installer.go

```diff
@@ -194,12 +194,15 @@ func (i *installer) finalize() error {
 		return err
 	}
 	if strings.Contains(out, ironicAgentServiceName) {
+		//update installation progress
+		i.UpdateHostInstallProgress(models.HostStageRebooting, "converged flow enabled, ironic will reboot the node shortly")
```

Comment:
```suggestion
		i.UpdateHostInstallProgress(models.HostStageRebooting, "Converged flow enabled, ironic will reboot the node shortly")
```

Reply:
This is user facing. I don't think most users (or even most people in our team) know what the hell converged flow means, it's not that interesting, so perhaps we should have a simpler message here?

Reply:
I lose the "converged flow enabled," part.